### PR TITLE
DOPS-171: Update translation process with weblate changes.

### PR DESCRIPTION
**Pull request by metanerd**
#### Summary
Minimalistic changes to keep the translation process up to date with the weblate migration. 

#### Ticket Link
https://mattermost.atlassian.net/browse/DOPS-171

### CODE REVIEW

File: source/developer/localization.rst [hunk 1]
```diff
@@ -15,20 +15,23 @@ To translate a language:
...
+5 - Review translation rules written by localization leads, when applicable:
 
     - `French (Règles pour la traduction francophone de Mattermost) <https://github.com/wget/mattermost-localization-french-translation-rules>`__
     - `German (Regeln zur Übersetzung von Mattermost) <https://gist.github.com/meilon/1317a9425988b3ab296c894a72270787>`__
 
 Translations Updates
 --------------------
 
-1 - New and updated strings are imported to the `Mattermost Translation Server <http://translate.mattermost.com>`__ each day at 00:00 GMT. Strings with a ``mobile`` prefix are imported from the `mattermost-mobile <https://github.com/mattermost/mattermost-mobile>`__ project, others are from `mattermost-server <https://github.com/mattermost/mattermost-server>`__ and `mattermost-webapp <https://github.com/mattermost/mattermost-webapp>`__.
+1 - New and updated source strings are being automatically pulled on github pushes by developers.
```
Comment (justinegeffen) [inline source/developer/localization.rst:34]:
```suggestion
1 - New and updated source strings are being automatically pulled on GitHub pushes by developers.
```

File: source/developer/localization.rst [hunk 2]
```diff
@@ -82,26 +85,22 @@ If you'd like to review and verify translations prior to achieving Beta-quality
 
 1 - Build Mattermost on your machine following the `Developer Machine Setup Guides <https://docs.mattermost.com/developer/dev-setup.html>`__.
 
-2 - Download a copy of your translations to your local machine.
-
-.. image:: ../../source/images/translations_download.png
-
-3 - Use `Mattermosti18n <https://github.com/rodrigocorsi2/mattermosti18n#convert-po---json>`__ to convert Pootle's output into JSON files.
+2 - Download a copy of your translations to your local machine by going to the language of your choice, e.g. `German <https://translate.mattermost.com/projects/mattermost/mattermost-server_master/de/>` -> Files -> Download original translation file (JSON file)
```
Comment (justinegeffen) [inline source/developer/localization.rst:88]:
```suggestion
2 - Download a copy of your translations to your local machine by going to the language of your choice, e.g. `German <https://translate.mattermost.com/projects/mattermost/mattermost-server_master/de/>` > Files > Download original translation file (``.json`` file).
```

File: source/developer/localization.rst [hunk 3]
```diff
@@ -82,26 +85,22 @@ If you'd like to review and verify translations prior to achieving Beta-quality
...
 
-4 - Copy the generated [locale].json files to the corresponding directories:
+3 - Copy the generated [locale].json files to the corresponding directories:
```
Comment (justinegeffen) [inline source/developer/localization.rst:90]:
```suggestion
3 - Copy the generated ``[locale].json`` files to the corresponding directories:
```

File: source/developer/localization.rst [hunk 4]
```diff
@@ -15,20 +15,23 @@ To translate a language:
...
     - `German (Regeln zur Übersetzung von Mattermost) <https://gist.github.com/meilon/1317a9425988b3ab296c894a72270787>`__
 
 Translations Updates
 --------------------
 
-1 - New and updated strings are imported to the `Mattermost Translation Server <http://translate.mattermost.com>`__ each day at 00:00 GMT. Strings with a ``mobile`` prefix are imported from the `mattermost-mobile <https://github.com/mattermost/mattermost-mobile>`__ project, others are from `mattermost-server <https://github.com/mattermost/mattermost-server>`__ and `mattermost-webapp <https://github.com/mattermost/mattermost-webapp>`__.
+1 - New and updated source strings are being automatically pulled on github pushes by developers.
 
 2 - A new pull request with latest translations that reach at least Beta Quality is submitted to the `mattermost-mobile <https://github.com/mattermost/mattermost-mobile>`__, `mattermost-server <https://github.com/mattermost/mattermost-server>`__ and `mattermost-webapp <https://github.com/mattermost/mattermost-webapp>`__ repos each Monday at 22:00 GMT.
```
Comment (justinegeffen) [inline source/developer/localization.rst:36]:
```suggestion
2 - A new pull request with latest translations that reach at least Beta quality is submitted to the `mattermost-mobile <https://github.com/mattermost/mattermost-mobile>`__, `mattermost-server <https://github.com/mattermost/mattermost-server>`__ and `mattermost-webapp <https://github.com/mattermost/mattermost-webapp>`__ repos each Monday at 22:00 GMT.
```

File: source/developer/localization.rst [hunk 5]
```diff
@@ -15,22 +15,25 @@ To translate a language:
...
+2 - Confirm that the language you want to help translate is listed in the shipped project of the `Translation Server <https://translate.mattermost.com/projects/mattermost/mattermost-server_master/>`__.
+If it is, perhaps offer to help translate or review and make suggestions.
 
-    - If it is, perhaps offer to help translate or review and make suggestions.
-    - If not, ask to setup your language on the `Mattermost localization channel <https://community.mattermost.com/core/channels/localization>`__, so you can start helping with the translation.
+3 - Confirm that the language you want to help translate is listed in the work in progress project of the `Translation Server <https://translate.mattermost.com/projects/i18n-wip/mattermost-server-wip/>`__.
+If it is, perhaps offer to help translate or review and make suggestions.
 
-3 - Review translation rules written by localization leads, when applicable:
+4 - The language you want to help translate is neither in the shipped project nor in the work in progress projects `projects <https://translate.mattermost.com/projects/>`__.
```
Comment (justinegeffen) [inline source/developer/localization.rst:24]:
```suggestion
4 - The language you want to help translate is neither in the shipped project nor in the work in progress `projects <https://translate.mattermost.com/projects/>`__.
```

**Comment by justinegeffen** [comment]
Some minor suggestions but otherwise LGTM! :)

**Comment by wget** [comment]
Discovered a few issues in the final version. I'm proposing changes. Please wait before merging.
+ integrating what has been discussed before from the L10n channel.

File: source/developer/localization.rst [hunk 6]
```diff
@@ -15,22 +15,25 @@ To translate a language:
 
 1 - Join the `Mattermost Translation Server <http://translate.mattermost.com>`__.
```
Comment (jasonblais) [inline source/developer/localization.rst:16]: Propose simplifying

```
1 - Join the `Mattermost Translation Server <http://translate.mattermost.com>`__.

2 - Confirm that the language you want to help translate is listed in the `Translation Server <https://translate.mattermost.com/projects/>`_. The language may be listed as shipped or work in progress, depending on if the language `ships in Mattermost <https://docs.mattermost.com/help/settings/account-settings.html#language>`_.

    - If it is, perhaps offer to help translate or review and make suggestions.
    - If not, ask to setup your language on the `Mattermost localization channel <https://community.mattermost.com/core/channels/localization>`__, so you can start helping with the translation.

3 - Review translation rules written by localization leads, when applicable:

    - `French (Règles pour la traduction francophone de Mattermost) <https://github.com/wget/mattermost-localization-french-translation-rules>`__
    - `German (Regeln zur Übersetzung von Mattermost) <https://gist.github.com/meilon/1317a9425988b3ab296c894a72270787>`__
```

File: source/developer/localization.rst [hunk 7]
```diff
@@ -15,22 +15,25 @@ To translate a language:
...
+5 - Review translation rules written by localization leads, when applicable:
 
     - `French (Règles pour la traduction francophone de Mattermost) <https://github.com/wget/mattermost-localization-french-translation-rules>`__
     - `German (Regeln zur Übersetzung von Mattermost) <https://gist.github.com/meilon/1317a9425988b3ab296c894a72270787>`__
 
 Translations Updates
 --------------------
 
-1 - New and updated strings are imported to the `Mattermost Translation Server <http://translate.mattermost.com>`__ each day at 00:00 GMT. Strings with a ``mobile`` prefix are imported from the `mattermost-mobile <https://github.com/mattermost/mattermost-mobile>`__ project, others are from `mattermost-server <https://github.com/mattermost/mattermost-server>`__ and `mattermost-webapp <https://github.com/mattermost/mattermost-webapp>`__.
+1 - New and updated source strings are being automatically pulled on GitHub pushes by developers.
```
Comment (jasonblais) [inline source/developer/localization.rst:34]: What does `being automatically pulled on GitHub pushes by developers` mean?

File: source/developer/localization.rst [hunk 8]
```diff
@@ -15,22 +15,25 @@ To translate a language:
...
 
-2 - A new pull request with latest translations that reach at least Beta Quality is submitted to the `mattermost-mobile <https://github.com/mattermost/mattermost-mobile>`__, `mattermost-server <https://github.com/mattermost/mattermost-server>`__ and `mattermost-webapp <https://github.com/mattermost/mattermost-webapp>`__ repos each Monday at 22:00 GMT.
+2 - A new pull request with latest translations that reach at least Beta quality is submitted to the `mattermost-mobile <https://github.com/mattermost/mattermost-mobile>`__, `mattermost-server <https://github.com/mattermost/mattermost-server>`__ and `mattermost-webapp <https://github.com/mattermost/mattermost-webapp>`__ repos each Monday at 22:00 GMT.
```
Comment (jasonblais) [inline source/developer/localization.rst:36]: Do we still have PRs submitted once per week on Mondays?

File: source/developer/localization.rst [hunk 9]
```diff
@@ -82,26 +85,22 @@ If you'd like to review and verify translations prior to achieving Beta-quality
 
 1 - Build Mattermost on your machine following the `Developer Machine Setup Guides <https://docs.mattermost.com/developer/dev-setup.html>`__.
 
-2 - Download a copy of your translations to your local machine.
-
-.. image:: ../../source/images/translations_download.png
-
-3 - Use `Mattermosti18n <https://github.com/rodrigocorsi2/mattermosti18n#convert-po---json>`__ to convert Pootle's output into JSON files.
+2 - Download a copy of your translations to your local machine by going to the language of your choice, e.g. `German <https://translate.mattermost.com/projects/mattermost/mattermost-server_master/de/>` > Files > Download original translation file (``.json`` file).
```
Comment (jasonblais) [inline source/developer/localization.rst:88]:
```suggestion
2 - Download a copy of your translations to your local machine by going to the language of your choice, e.g. `German <https://translate.mattermost.com/projects/mattermost/mattermost-server_master/de/>`_, then **Files > Download original translation file**.
```

File: source/developer/localization.rst [hunk 10]
```diff
@@ -130,7 +129,7 @@ If you're interested in contributing to the process, please join the `Mattermost
     "Türkçe - Turkish", "`Kaya Zeren <https://twitter.com/kaya_zeren>`__", ""
     "Pусский - Russian", "`Alexey Napalkov <https://github.com/flynbit>`_", ""
     "Yкраїнська - Ukrainian", "TBD (Open Role)", ""
-    "한국어 - Korean", "TBD (Open Role)", ""
+    "한국어 - Korean", "`avasconcelos114 <https://translate.mattermost.com/user/avasconcelos114/>`__", ""
```
Comment (jasonblais) [inline source/developer/localization.rst:132]: Confirming Andre agreed to be the official reviewer/translator for Korean?

File: source/developer/localization.rst (hunk 8 above)
Comment (metanerd) [reply source/developer/localization.rst:36]: With some new changes I did, it should.

File: source/developer/localization.rst (hunk 10 above)
Comment (metanerd) [reply source/developer/localization.rst:132]: Actually no, you are right. Removed.

File: source/developer/localization.rst (hunk 8 above)
Comment (jasonblais) [reply source/developer/localization.rst:33]: Awesome!

File: source/developer/localization.rst [hunk 11]
```diff
@@ -28,9 +28,9 @@ To translate a language:
 Translations Updates
 --------------------
 
-1 - New and updated strings are imported to the `Mattermost Translation Server <http://translate.mattermost.com>`__ each day at 00:00 GMT. Strings with a ``mobile`` prefix are imported from the `mattermost-mobile <https://github.com/mattermost/mattermost-mobile>`__ project, others are from `mattermost-server <https://github.com/mattermost/mattermost-server>`__ and `mattermost-webapp <https://github.com/mattermost/mattermost-webapp>`__.
+1 - New and updated source strings are being automatically added to the translation server on GitHub pushes by developers.
```
Comment (jasonblais) [inline source/developer/localization.rst:31]: @metanerd To confirm, do you mean new and updated source strings are automatically added to the translation service each time a relevant GitHub PR is merged?

Comment (metanerd) [reply source/developer/localization.rst:31]: Yes, I do mean exactly that.

Comment (metanerd) [reply source/developer/localization.rst:31]: Should I use your phrasing?

**Comment by metanerd** [comment]
> @metanerd Would you be open to add instructions for how to update user permissions in Weblate? I tried but was unsuccessful and the docs didn't prove to be useful.

@jasonblais added.

File: source/developer/localization.rst (hunk 11 above)
Comment (jasonblais) [reply source/developer/localization.rst:31]: @metanerd yes, lets

Comment (jasonblais) [inline source/developer/localization.rst:31]:
```suggestion
1 - New and updated source strings are automatically added to the translation server each time a GitHub PR containing source strings is merged.
```

**Comment by jasonblais** [comment]
@justinegeffen Ready for re-review, pending feedback from @wget :)

File: source/developer/localization.rst [hunk 12]
```diff
@@ -134,3 +130,8 @@ If you're interested in contributing to the process, please join the `Mattermost
     "中文 (简体) - Simplified Chinese", "`aeomin <http://translate.mattermost.com/user/aeomin/>`__", ""
     "中文 (繁體) - Traditional Chinese", "`Tze-Kei Lee (chikei) <https://github.com/chikei>`__", ""
     "日本語 - Japanese", "`Yusuke Nemoto (kaakaa) <https://github.com/kaakaa>`__", ""
+
+Administrative tasks
+~~~~~~~~
+To grant trusted translators additional permissions as weblate admin, add the user to the mattermost@TrustedReviewers Group in weblate.
```
Comment (justinegeffen) [inline source/developer/localization.rst:136]:
```suggestion
To grant trusted translators additional permissions as Weblate admin, add the user to the *mattermost@TrustedReviewers* group in Weblate.
```

File: source/developer/localization.rst [hunk 13]
```diff
@@ -134,3 +130,8 @@ If you're interested in contributing to the process, please join the `Mattermost
...
+Admin interface (tool icon in weblate) > Users > Django Admin Interface > click on the User to add to the group > Groups > Add to mattermost@TrustedReviewers > Save.
```
Comment (justinegeffen) [inline source/developer/localization.rst:137]:
```suggestion
**Admin interface (tool icon in Weblate) > Users > Django Admin Interface > select the User > Groups > Add to *mattermost@TrustedReviewers* > Save**.
```

File: source/developer/localization.rst [hunk 14]
```diff
@@ -82,26 +82,22 @@ If you'd like to review and verify translations prior to achieving Beta-quality
...
     - For the server, copy the files to the i18n directory of the ``mattermost-server`` project.
     - For the webapp, copy the files to the i18n directory of the ``mattermost-webapp`` project.
 
-5 - Modify the file ``i18n/i18n.jsx`` in the ``mattermost-webapp`` project to include your translated strings.
+4 - Modify the file ``i18n/i18n.jsx`` in the ``mattermost-webapp`` project to include your translated strings.
 
-6 - Compile and run **Mattermost** to confirm everything works. You can then review and verify translations from your machine.
+5 - Compile and run **Mattermost** to confirm everything works. You can then review and verify translations from your machine.
 
 If you find a string that has not been translated, search for the string in the respective localization file to confirm it's included for translations. You can find the English version for server, webapp and mobile projects below
```
Comment (justinegeffen) [inline source/developer/localization.rst:96]:
```suggestion
If you find a string that has not been translated, search for the string in the respective localization file to confirm it's included for translations. You can find the English version for server, webapp, and mobile projects below
```

File: source/developer/localization.rst [hunk 15]
```diff
@@ -15,7 +15,7 @@ To translate a language:
 
 1 - Join the `Mattermost Translation Server <http://translate.mattermost.com>`__.
 
-2 - Confirm that the language you want to help translate is listed in the `Translation Server <http://translate.mattermost.com>`__.
+2 - Confirm that the language you want to help translate is listed in the `Translation Server <https://translate.mattermost.com/projects/>`_. The language may be listed as shipped or work in progress, depending on if the language `ships in Mattermost <https://docs.mattermost.com/help/settings/account-settings.html#language>`_.
```
Comment (justinegeffen) [inline source/developer/localization.rst:18]:
```suggestion
2 - Confirm that the language you want to help translate is listed in the `Translation Server <https://translate.mattermost.com/projects/>`_. The language may be listed as shipped or work in progress, depending whether the language `ships in Mattermost <https://docs.mattermost.com/help/settings/account-settings.html#language>`_.
```

File: source/developer/localization.rst [hunk 16]
```diff
@@ -28,9 +28,9 @@ To translate a language:
...
+1 - New and updated source strings are being automatically added to the translation server on GitHub pushes by developers.
 
-2 - A new pull request with latest translations that reach at least Beta Quality is submitted to the `mattermost-mobile <https://github.com/mattermost/mattermost-mobile>`__, `mattermost-server <https://github.com/mattermost/mattermost-server>`__ and `mattermost-webapp <https://github.com/mattermost/mattermost-webapp>`__ repos each Monday at 22:00 GMT.
+2 - A new pull request with latest translations that reach at least Beta quality is submitted to the `mattermost-mobile <https://github.com/mattermost/mattermost-mobile>`__, `mattermost-server <https://github.com/mattermost/mattermost-server>`__ and `mattermost-webapp <https://github.com/mattermost/mattermost-webapp>`__ repos each Monday at 22:00 GMT.
```
Comment (justinegeffen) [inline source/developer/localization.rst:33]:
```suggestion
2 - A new pull request with latest translations that reach at least Beta quality is submitted to the `mattermost-mobile <https://github.com/mattermost/mattermost-mobile>`__, `mattermost-server <https://github.com/mattermost/mattermost-server>`__, and `mattermost-webapp <https://github.com/mattermost/mattermost-webapp>`__ repos each Monday at 22:00 GMT.
```

File: source/developer/localization.rst (hunk 11 above)
Comment (wget) [reply source/developer/localization.rst]: Added

File: source/developer/localization.rst (hunk 7 above)
Comment (wget) [reply source/developer/localization.rst]: Same. Added.

**Pull request merged**